configs/messages.go

```diff
@@ -0,0 +1,22 @@
+package configs
+
+// All the strings that are needed for debugging, info and error logging.
+const (
+	CheckingDependencies          = "Checking dependencies: %s"
+	DependenciesPending           = "Pending dependencies: %s"
+	DependencyNotInstalled        = "Dependency %s is not installed on host machine"
+	DependenciesOK                = "All dependencies are installed on host machine"
+	GeneratingDockerComposeScript = "Generating docker-compose script for current selection of clients"
+	InstallingDependenciesError   = "Something went wrong while installing dependencies. Error: %s"
+)
+
+//TODO: Move all this to a config file. No supported clients for now
+
+var Eth1ClientsSupported = []string{}
```

Comment:
variable naming convention to be followed

Reply:
This variable is hardcoded here for now. Will move this stuff to a config file in a way we need to discuss. For now, it must be uppercased because it needs to be exported

Reply:
Bundle it together in a struct then
struct called config